src/S1_fft_cc_MPI.py

```diff
@@ -413,6 +427,9 @@
 print('it takes %6.2fs to process step 1 in total' % (tt1-tt0))
 comm.barrier()
 
+sfile = rootpath+ '/CCF/2016_07_01_00_00_00T2016_07_02_00_00_00.h5'
```

Comment:
Should this  be `cc_h5` from lines 375:389?
```
cc_h5 = os.path.join(CCFDIR,tname)
```
In general this hard-coded path will fail